node/src/components/fetcher/metrics.rs

```diff
@@ -3,7 +3,7 @@ use prometheus::{IntCounter, Registry};
 use crate::unregister_metric;
 
 #[derive(Debug)]
-pub(crate) struct FetcherMetrics {
+pub(crate) struct Metrics {
```

Comment:
```suggestion
pub(super) struct Metrics {
```

Reply:
Actually, on the `feat-fast-sync` the `Fetcher` further exposes the `Metrics` type via the public `ItemFetcher` trait.

I'll see if I can tune the code structure in order not to expose `Metrics` on the crate level.

Reply:
As agreed on the call, we'll merge as-is.